Patch Makefile to add optimisation flag and produce position-independent code
Simon Pinches

diff -Nru libdierckx-master-orig/Makefile libdierckx-master/Makefile
--- libdierckx-master-orig/Makefile	2016-05-11 14:03:58.000000000 +0200
+++ libdierckx-master/Makefile	2021-08-04 16:18:54.090269000 +0200
@@ -87,7 +87,7 @@
 FC = gfortran
 OBJEXT = o
 RANLIB = ranlib
-FCFLAGS = -fdefault-real-8 -fdefault-double-8 -g -Wall
+FCFLAGS = -fdefault-real-8 -fdefault-double-8 -g -Wall -O2 -fPIC
 
 all: libdierckx.a 
 
